data_examine/generate_continuations.py

```diff
@@ -0,0 +1,70 @@
+import datasets
```

Comment:
There is a similar file called `infer.py` in another pr which does multi-gpu inference. We should choose one as a default or combine them